ansible/group_vars/all

```diff
@@ -25,6 +25,9 @@ controller:
   port: 10001
   blackboxFraction: 0.10
 
+invokerHealth:
+  testAction: whisk.system/utils/date
```

Comment:
Reasoning is, that having "real" payload is preferable here but without loosing user activations, hence this simulates real load as close as possible which I think is desirable to catch whatever might be wrong with the invoker.